platform/darwin/src/MGLFillStyleLayer.h

```diff
@@ -13,6 +13,14 @@ typedef NS_ENUM(NSUInteger, MGLFillStyleLayerFillTranslateAnchor) {
 
 @interface MGLFillStyleLayer : MGLBaseStyleLayer <MGLStyleLayer>
 
+/**
+ A predicate that corresponds to the layer's <a href='https://www.mapbox.com/mapbox-gl-style-spec/#types-filter'>filter</a>.
+ 
+ The predicate's left expression must be a string that identifies a feature
+ property, or one of the special keys.
```

Comment:
Oh, I’m seeing now that this documentation was originally inline but was removed in response to https://github.com/mapbox/mapbox-gl-native/pull/6049#discussion_r76650575. I meant that we should describe the `$type` and `$id` keys but do so in an HTML definition list.